src/cdrizzlemap.c

```diff
@@ -442,6 +457,38 @@ is_point_strictly_in_hp(const struct vertex pt, const struct vertex v_,
     return ((area(v, pt) - area(v_, pt) - area(v, v_)) > 0.0);
 }
 
+/**
+ * Rotates the vertices of a polygon by a specified number of positions.
+ *
+ * This function shifts the vertices of the given polygon `p` by `pos`
+ * positions. For each rotation, the first vertex is moved to the end of the
+ * vertex array, effectively rotating the polygon's vertex order. The operation
+ * is performed in-place and repeated `pos` times.
+ *
+ * @param p   Pointer to the polygon structure to be rotated. The polygon is
+ *            expected to have an array of vertices `v` and a count `npv`.
+ * @param pos Number of positions to rotate the polygon's vertices.
+ */
+void
+rotate_polygon(struct polygon *p, int pos) {
```

Comment:
Is this used anywhere?

Reply:
no, it is not used unless I need to debug stuff: then this comes in handy